src/tests/unit/test_tree_functionality/get_node_for_tree_tests.py

```diff
@@ -0,0 +1,142 @@
+from domain_classes.blueprint_attribute import BlueprintAttribute
+from domain_classes.tree_node import ListNode, Node
+from enums import REFERENCE_TYPES, SIMOS
+from tests.unit.mock_utils import (
+    get_mock_document_service,
+    mock_storage_recipe_provider,
+)
+from tests.unit.test_tree_functionality.blueprints_for_tree_tests import get_blueprint
+
+
+def get_engine_package_node() -> Node:
```

Comment:
content of get_engine_package_node() is the same as in the old get_engine_package_node() inside the old file "test_tree_node_helpers.py"